test/activerecord_spanner_adapter/table/column_test.rb

```diff
@@ -42,8 +42,8 @@ def test_set_default_nullable_for_non_primary_key_column
   def test_set_default_not_nullable_for_primary_key_column
     column = new_table_column
     column.primary_key = true
-    assert_equal column.primary_key, true
-    assert_equal column.nullable, false
+    assert_equal true, column.primary_key
```

Comment:
In this case the order was wrong. The signature of `assert_equal` is: `def assert_equal exp, act, msg = nil`

I noted it because in this case the test initially failed, and I got confused as the expected and actual values were in the wrong order. There are actually multiple places where this is wrong, so we should probably fix that in one large PR. The reason I changed it here was (apart from the fact that it initially failed), at least one other line in the test was changed.